files/config/puppet-inet-filter.nft

```diff
@@ -1 +1,2 @@
+  include "inet-filter-helper-*.nft"
```

Comment:
Can't recall it from the top of my head, but I guess that is fine, even if that expands to 0 files - if no helper is defined?

Reply:
correct, similar to other optional resources:
```
# grep include /etc/nftables/puppet.nft 
include "file-*.nft"
include "custom-*.nft"
include "inet-filter.nft"

```